modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/dto/v1_0/converter/UserResourceDTOConverter.java

```diff
@@ -106,7 +106,7 @@ public UserAccount toDTO(DTOConverterContext dtoConverterContext, User user)
 			return null;
 		}
 
-		Contact contact = user.getContact();
+		Contact contact = user.fetchContact();
```

Comment:
I don't think it's the right solution because:
1) why the user has null contact? In the creation of a new user it should create always the contact
2) if this is null I think that the below ```contact.getPrefixListTypeId``` will throw NPE